R/utilities-units.R

```diff
@@ -662,6 +662,14 @@ ospUnits <- list()
 #' @keywords internal
 #' @noRd
 .yErrorUnitConverter <- function(yData, yTargetUnit) {
+  # If error type is geometric, conversion of `yValues` to different units
+  # should not trigger conversion of error values (and units)
+  if ("yErrorType" %in% colnames(yData) &&
+    !is.na(unique(yData$yErrorType)) &&
+    unique(yData$yErrorType) == "GeometricStdDev") {
```

Comment:
Please use the enum value `DataErrorType$GeometricStdDev`.

Reply:
Done!